components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.machinelearner.ui/pom.xml

```diff
@@ -0,0 +1,92 @@
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <groupId>org.wso2.carbon.mediation</groupId>
+        <artifactId>mediators-ui</artifactId>
+        <version>4.3.3-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.carbon.mediator.machinelearner.ui</artifactId>
+    <version>4.3.3-SNAPSHOT</version>
+    <packaging>bundle</packaging>
+    <name>WSO2 Carbon - Machine Learner Mediator UI</name>
+    <url>http://wso2.org</url>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Bundle-Activator>org.wso2.carbon.mediator.machinelearner.ui.MLMediatorActivator</Bundle-Activator>
+                        <Carbon-Component>UIBundle</Carbon-Component>
+                        <Export-Package>
+                            org.wso2.carbon.mediator.machinelearner.ui.*,
+                            !javax.servlet,
+                            !javax.servlet.http,
+                        </Export-Package>
+                        <Import-Package>
+                            !javax.xml.namespace,
+                            javax.xml.namespace; version=0.0.0,
+                            org.apache.axiom.om; version="${axiom.osgi.version.range}",
+                            org.wso2.carbon.mediator.service; version="${carbon.mediation.imp.pkg.version}",
+                            org.wso2.carbon.mediator.service.ui; version="${carbon.mediation.imp.pkg.version}",
+                            *;resolution:=optional
+                        </Import-Package>
+
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.synapse</groupId>
+            <artifactId>synapse-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.axis2</groupId>
+                    <artifactId>axis2-codegen</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.mediator.service</artifactId>
+            <version>4.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.logging</artifactId>
+            <version>4.3.0</version>
```

Comment:
Please move the version numbers from this module to the parent pom of the repository. Most of these references would be in the parent pom. No need to add the version number here. Please do the same for the remaining dependencies.